### PR TITLE
Add sysconfig for OIST deigo

### PR DESCRIPTION
**Pull request by tristan0x**

### CODE REVIEW

File: sysconfig/deigo/compilers.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+compilers:
+- compiler:
+    spec: clang@8.0.1
+    paths:
+      cc: /usr/bin/clang-8
+      cxx: /usr/bin/clang++-8
+      f77:
+      fc:
```
Comment (pramodk) [inline sysconfig/deigo/compilers.yaml]: in case people want to use clang, we typically add `gfortran` here.

File: sysconfig/deigo/modules.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      - PKG_CONFIG_PATH
+    lib64/pkgconfig:
+      - PKG_CONFIG_PATH
+    '':
+      - CMAKE_PREFIX_PATH
+  tcl:
+    all:
+      autoload: 'all'
+      suffixes:
+          '^python@2.7.12': 'python2'
```
Comment (pramodk) [inline sysconfig/deigo/modules.yaml]: is there leftover or is this python2 exist?

**Review by pramodk — DISMISSED** [review]
lgtm

**Review by matz-e — APPROVED** [review]
LGTM, note to self that some of these config files need to be updated in the next deployment (config file format change)

**Pull request merged**